app/services/fuzzySearchService.js

```diff
@@ -102,6 +102,13 @@ const groupByPayee = (searchResults) => {
   return result
 }
 
+const getSearchSuggestions = async (searchKey) => {
+  const paymentData = await getAllPaymentData()
+  const fuse = new Fuse(paymentData, fuseSearchOptions)
+  const searchResult = fuse.search(searchKey).map(row => row.item)
```

Comment:
Have you tested this? I think a simple search will result in same payee being returned twice if that payee has received payment for both schemes.

Reply:
If you look at the getPaymentData function, we call groupByPayee function to get bundle same payees together

Reply:
good point, the groupby is applied now.